### DISCUSSION TITLE
Is it possible to Ping via console?

### DISCUSSION THREAD

**Discussion by zeroarst** [arendst/Tasmota#21222] [Q&A]
I would like to test if my tasmota can connect to internet or other devices. I checked the document it says I need `#define USE_PING`. I am wondering if this is possible via console? Or is there other way to test if tasmota can send mqtt to my mosquitto broker?

**Comment by sfromis** [comment]
As long as you use a build including `#define USE_PING`, you have the `Ping` command available similar to all other commands, including the console. However, `Ping` is not useful for checking MQTT connectivity and setup, as factors potentially hindering that are more complex that checking for IP connectivity. No features for verifying MQTT without actually trying.
